server/src/main/java/org/elasticsearch/threadpool/FixedExecutorBuilder.java

```diff
@@ -49,6 +50,25 @@ public final class FixedExecutorBuilder extends ExecutorBuilder<FixedExecutorBui
         this(settings, name, size, queueSize, "thread_pool." + name, trackExecutionTime);
     }
 
+    /**
+     * Construct a fixed executor builder; the settings will have the key prefix "thread_pool." followed by the executor name.
+     *
+     * @param settings  the node-level settings
+     * @param name      the name of the executor
+     * @param size      the fixed number of threads
+     * @param queueSize the size of the backing queue, -1 for unbounded
+     * @param taskTrackingConfig whether to track statics about task execution time
+     */
+    FixedExecutorBuilder(
```

Comment:
Similar to my other comment, there are only 19 uses of the larger ctor (taking more params). Can we avoid adding yet-another-ctor by adjusting the more full fledged ctor?